modules/installation/proc-install-rhdh-osd-gcp-operator.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+// assembly-install-rhdh-osd-gcp.adoc
+
+[id="proc-install-rhdh-osd-gcp-operator_{context}"]
+= Installing {product} on {osd-short} on {gcp-short} using the Operator
+
+You can install {product-short} on {osd-short} on {gcp-short} using the {product} Operator.
+
+.Prerequisites
+* You have a valid {gcp-short} account.
+* Your {osd-short} cluster is running on {gcp-short}. For more information, see link:https://docs.redhat.com/en/documentation/openshift_dedicated/4/html/installing_accessing_and_deleting_openshift_dedicated_clusters/osd-creating-a-cluster-on-gcp[Creating a cluster on GCP] in {osd-brand-name} documentation.
+* You have administrator access to {osd-short} cluster and {gcp-short} project.
+
+.Procedure
+
+. Access your {osd-short} cluster using the {ocp-short} web console.
+. In the *Administrator* perspective of the {ocp-short} web console, click *Operators > OperatorHub*.
+. In the *Filter by keyword* box, enter {product-short} and click the *{product} Operator* card.
+. On the *{product} Operator* page, click *Install*.
+. In the {ocp-short} console, navigate to *Installed Operators* and select *{product} Operator*.
+. From the {product-short} Operator page, click *Create New Instance*.
+. Specify the name and namespace where you want to deploy {product-short}.
+. Configure the required settings such as Git integration, secret management, and user permissions.
+. Choose deployment options based on your organization’s requirements.
+. Review the configuration and click *Create*.
+
+.Verification
+
+To access the {product-short}, navigate to the {product-short} URL provided in the {ocp-short} web console.
+
+.Additional resources
+* link:{installing-on-ocp-book-url}[{installing-on-ocp-book-title}]
+* link:{installing-on-eks-book-url}[{installing-on-eks-book-title}]
+* link:{installing-on-aks-book-url}[{installing-on-aks-book-title}]
```

Comment:
If I have already chosen my platform, links to other platform installation methods are probably less relevant than links to the administration guide.

Other installation guides have no Additional resources sections, you might want to omit it entirely.

Reply:
It’s a good practice to include additional resources in the documentation. Instead of removing the additional resources, I'll add admin guide link.